src/state/queries/util.ts

```diff
@@ -0,0 +1,17 @@
+import {QueryClient, QueryKey, InfiniteData} from '@tanstack/react-query'
+
+export function truncateAndInvalidate<T = any>(
```

Comment:
Any?

Reply:
Maybe doesn't matter since it's all infinite huh

Reply:
yeah we dont touch that part of the object